src/clarity-angular/data/datagrid/datagrid.ts

```diff
@@ -38,8 +39,18 @@ import {DatagridRenderOrganizer} from "./render/render-organizer";
     selector: "clr-datagrid",
     templateUrl: "./datagrid.html",
     providers: [
-        Selection, Sort, FiltersProvider, Page, Items, DatagridRenderOrganizer, RowActionService, ExpandableRowsCount,
-        HideableColumnService, StateDebouncer, StateProvider
+        Selection,
+        Sort,
+        FiltersProvider,
+        Page,
+        Items,
+        DatagridRenderOrganizer,
+        RowActionService,
+        ExpandableRowsCount,
+        HideableColumnService,
+        StateDebouncer,
+        StateProvider,
+        ColumnToggleButtonsService,
```

Comment:
Is this needed here? I can't find where it is used.

Reply:
Its used in child components, but needs to be here so it can be provided.